src/middlewares/functionShield.js

```diff
@@ -0,0 +1,22 @@
+const functionShield = require('@puresec/function-shield')
+const merge = require('deepmerge')
+
+module.exports = (opts = {}) => ({
+  before: (handler, next) => {
+    const defaults = {
+      policy: {
+        outbound_connectivity: 'block',
+        read_write_tmp: 'block',
+        create_child_process: 'block',
+        read_handler: 'block'
+      },
+      disable_analytics: false,
+      token: process.env.FUNCTION_SHIELD_TOKEN || handler.context.FUNCTION_SHIELD_TOKEN
+    }
+    const options = merge(defaults, opts)
+
+    functionShield.configure(options)
```

Comment:
this would run `functionShield.configure` on every invocation, how heavy is this call? 

is it worth doing it on every invocation? would it be enough to just do it during coldstart?